packages/core/src/utils/defaults.ts

```diff
@@ -0,0 +1,66 @@
+import { type Address, type Chain } from "viem";
+import {
+  arbitrum,
+  arbitrumGoerli,
+  base,
+  baseGoerli,
+  goerli,
+  mainnet,
+  optimism,
+  optimismGoerli,
+  polygon,
+  polygonMumbai,
+  sepolia,
+} from "viem/chains";
+
+/**
+ * Utility method returning the entry point contrafct address given a {@link Chain} object
+ *
+ * @param chain - a {@link Chain} object
+ * @returns a {@link abi.Address} for the given chain
+ * @throws if the chain doesn't have an address currently deployed
+ */
+export const getDefaultEntryPointContract = (chain: Chain): Address => {
+  switch (chain.id) {
+    case mainnet.id:
+    case sepolia.id:
+    case goerli.id:
+    case polygon.id:
+    case polygonMumbai.id:
+    case optimism.id:
+    case optimismGoerli.id:
+    case arbitrum.id:
+    case arbitrumGoerli.id:
+    case base.id:
+    case baseGoerli.id:
+      return "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+  }
+  throw new Error("could not find entry point contract address");
+};
+
+/**
+ * Utility method returning the default simple account factory address given a {@link Chain} object
+ *
+ * @param chain - a {@link Chain} object
+ * @returns a {@link abi.Address} for the given chain
+ * @throws if the chain doesn't have an address currently deployed
+ */
+export const getDefaultSimpleAccountFactory = (chain: Chain): Address => {
```

Comment:
nit(naming): to me the method name implies retrieving an object containing details about a factory, not just the address.

should we name it: `getDefaultSimpleAccountFactoryAddress`